config/webpack.js

```diff
@@ -0,0 +1,81 @@
+/* eslint-disable quote-props */
```

Comment:
Only for this file. But the error was on a JSON file (package.json), and JSON does not accept comments.